frappe/patches/v13_0/increase_password_length.py

```diff
@@ -1,7 +1,4 @@
 import frappe
 
 def execute():
-	frappe.db.multisql({
-		"mariadb": "ALTER  TABLE `__Auth` MODIFY `password` TEXT NOT NULL",
-		"postgres": 'ALTER TABLE "__Auth" ALTER COLUMN "password" TYPE TEXT'
-	})
+	frappe.db.sql(frappe.qb.change_table_type(tb = "__Auth",col = "password",type = "TEXT"))
```

Comment:
Method and kwarg names can be improved.

Coud be called change column type...also moved into the ORM instead.